Comments/ClientApp/src/context/UserContext.js

```diff
@@ -29,6 +33,7 @@ export class UserProvider extends React.Component<PropsType, StateType> {
 			displayName: "",
 			signInURL: "",
 			registerURL: "",
+			updateContext: this.updateContext,
```

Comment:
```suggestion
			updateContext: this.updateContext,
```
Should this be in the state type declaration object above?